nl.hjdskes.gcolor3.json

```diff
@@ -24,7 +24,13 @@
                 {
                     "type": "git",
                     "url": "https://github.com/flatpak/libportal.git",
-                    "commit": "bff3289781ef27388e75f772d5fe9dee20fdfb53"
+                    "commit": "bff3289781ef27388e75f772d5fe9dee20fdfb53",
+                    "tag": "0.4",
+                     "commit": "f68764e288ede516d902b131cc4fadded3804059",
+                     "x-checker-data": {
+                         "type": "git",
+                         "tag-pattern": "^([\\d.]+)$"
+                     }
```

Comment:
for the record this was a bad idea. 0.5 is API incompatible. I'm reverting this.

Reply:
You haven't returned yet. Do it to me?

Reply:
my bad I thought i had pushed some changes and I didn't.

Will do.

Reply:
this is done now.

Reply:
via #13